bikespace_frontend/gatsby-config.ts

```diff
@@ -24,7 +24,7 @@ const config: GatsbyConfig = {
       "path": "./src/images/"
     },
     __key: "images"
-  }, "gatsby-plugin-vanilla-extract"]
+  }, "gatsby-plugin-vanilla-extract", "gatsby-plugin-sass"]
```

Comment:
I think we need this package set in the `package.json` to run, it's complaining it doesn't have this package when it try to run.

Reply:
oh! yeah I found didn't go to the sub-folder to do `npm install`. It's done